test/benchmarks/driver_bench/src/main.mts

```diff
@@ -198,6 +199,33 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
   return [...results, ...compositeResults];
 }
 
-const finalResults = calculateCompositeBenchmarks(results);
+function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
+  const primesBench = results.find(r => r.info.test_name === 'primes');
+  const pingBench = results.find(r => r.info.test_name === 'ping');
+
+  if (pingBench) {
+    const pingThroughput = pingBench.metrics[0].value;
+    for (const bench of results) {
+      if (bench.info.test_name === 'ping') {
```

Comment:
pingBench is === to bench inside this if stmt right? so what's this loop really for?